### DISCUSSION TITLE
Mobile Codespaces

### DISCUSSION THREAD

**Discussion by Trawmoney** [community/community#50085] [Mobile]
Will codespaces ever be available for mobile? Replit is pretty amazing, but mobile still needs IDE

**Comment by stevepopovich** [comment]
Hi @Trawmoney, we currently don't have codespaces planned for mobile, but we will take it into consideration. 

FYI we have code editing on mobile widely available!

https://github.blog/changelog/2022-09-13-github-for-ios-edit-files-in-pull-requests/
https://github.blog/2023-03-07-file-editing-on-github-mobile-keeps-leveling-up/

Thanks!

**Reply by Eveeifyeve** [reply]
I would like to see it planned as it would be pretty cool. Like opening a code space on an Ipad from the app with official support without going to https://github.com/codespaces every time you want to code from your ipad.

**Reply by Eveeifyeve** [reply]
I have mentioned this discussion in #78118

**Reply by IdanAdar** [reply]
@stevepopovich You can edit on mobile, but if your mobile happens to be an iPad, you get a tiny edit window instead of making use of the iPad screen size -- please see my separate discussion for this topic. 🙏
